pkgs/applications/networking/cluster/hadoop/default.nix

```
@@ -51,46 +69,54 @@ let
           computers, each of which may be prone to failures.
         '';
         maintainers = with maintainers; [ volth illustris ];
-        platforms = [ "x86_64-linux" ];
+        platforms = builtins.attrNames sha256;
```

```suggestion
        platforms = attrNames sha256;
```

since there is a `with lib` at the start, using `lib.attrNames` is simpler.